runners/mlbox_docker_run/mlbox_docker_run/__main__.py

```diff
@@ -0,0 +1,35 @@
+import os
+import typer
+from mlcommons_box import parse   # Do not remove (it registers schemas on import)
+from mlcommons_box.common import mlbox_metadata
+from mlbox_docker_run import metadata
+from mlbox_docker_run.docker_run import DockerRun
+
+
+app = typer.Typer()
```

Comment:
Same comment as above with the usage of Click over Typer. Again, so sorry for the inconvenience